bubble_sort.rb

```diff
@@ -0,0 +1,36 @@
+def bubble_sort(array)
+
+  size = array.length - 1
+  loop do
+    swapped = false
+    0.upto(size).each do |n|
```

Comment:
its more efficient written like this `(size).times do `

Reply:
Oh, yeah, since it starts from 0, right? Haha, how dumb of me! Thank you for pointing that!